docker/server/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.6.2
+
+ENV PYTHONUNBUFFERED 1
+
+# Setup Debian linux
+RUN export DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get -y install build-essential curl
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
+RUN apt-get install -y nodejs
+RUN nodejs -v && npm -v
```

Comment:
You might want to clean up after running apt-get `rm -rf /var/lib/apt/lists/*` 
reference: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run